src/libraries/Common/tests/System/Xml/XmlDiff/XmlDiffDocument.cs

```diff
@@ -1550,15 +1560,19 @@ public class XmlDiffCharacterData : XmlDiffNode
     {
         private string _value;
         private XmlDiffNodeType _nodetype;
-        public XmlDiffCharacterData(string value, XmlDiffNodeType nt, bool NormalizeNewline)
+        public XmlDiffCharacterData(string value, XmlDiffNodeType nt, bool NormalizeNewline, bool NormalizeSpaces, bool IgnoreWhitespace)
             : base()
         {
-            this._value = value;
             if (NormalizeNewline)
-            {
-                this._value = this._value.Replace("\n", "");
-                this._value = this._value.Replace("\r", "");
-            }
+                value = value.Replace("\n", "").Replace("\r", "");
```

Comment:
>"\n" [](http://example.com/codeflow?start=38&length=4)

use character instead of string here? `'\n'` and `'\r'`

Reply:
We're* completely stripping the CR and LF out entirely, can't do a `String.Replace(char, char)` here can we? What would the replacement character be?

(*this is preexisting code)

Reply:
never mind, somehow, I saw the empty string as space.